specs/ingestion/common/taskParameters.yml

```
@@ -70,6 +70,14 @@ triggerType:
       $ref: './schemas/task.yml#/TriggerType'
     example: onDemand,schedule,subscription,streaming
 
+withNotifications:
+  name: withNotifications
```

```suggestion
withEmailNotifications:
  name: withEmailNotifications
```

better be descriptive than introduce bc when we add more notifications